src/index.js

```diff
@@ -1,8 +1,10 @@
 import 'babel-polyfill';
 import SimpleImgProvider from './simpleImgProvider';
 import SimpleImg from './simpleImg';
+import init from './init';
 
 export {
+  init,
```

Comment:
Maybe rename `init` to `initialiseRsi` so it's more readable in the consumer's code.

Reply:
how about `initSimpleImg`?